index.js

```diff
@@ -123,6 +123,8 @@ Cabal.prototype.publishNick = function (nick, cb) {
 }
 
 Cabal.prototype.publishChannelTopic = function (channel, topic, cb) {
+  if (!channel || typeof channel !== 'string') return cb()
+  if (!topic || typeof topic !== 'string') return cb()
```

Comment:
do you think it makes more sense to return errors in these cases? otherwise callers may think that the publish succeeded.